doc/manual.rst

```diff
@@ -7062,8 +7062,10 @@ one can import C++'s templates rather easily without the need for a pattern
 language for object types:
 
 .. code-block:: nim
+  :test: "nim cpp $1"
```

Comment:
warning as errors may be useful in manual tests

Reply:
https://github.com/timotheecour/Nim/issues/64#issuecomment-751982665